src/index_template.html

```diff
@@ -22,10 +44,11 @@
     <!-- <link rel="stylesheet" type="text/css" href="/geoplatform-demos/assets/css/reset.css" />
     <link rel="stylesheet" type="text/css" href="/geoplatform-demos/assets/css/code-styler.css" />
     <link rel="stylesheet" type="text/css" href="/geoplatform-demos/assets/css/gh-pages-overrides.css" /> -->
-    <link rel="stylesheet" type="text/css" href="#PUBLIC_URL#/assets/css/gp-external.min.css" />
+    <link rel="stylesheet" type="text/css" href="https://sit.geoplatform.info/assets/css/gp-external.min.css" />
   
-    <link rel="icon" type="image/x-icon" href="#PUBLIC_URL#/assets/images/web-icons/favicon.ico" />
+    <link rel="icon" type="image/x-icon" href="https://sit.geoplatform.info/assets/images/web-icons/favicon.ico" />
```

Comment:
47 and 49 need to be left as

`<link rel="stylesheet" type="text/css" href="#PUBLIC_URL#/assets/css/gp-external.min.css" />`

`<link rel="icon" type="image/x-icon" href="#PUBLIC_URL#/assets/images/web-icons/favicon.ico" />`